### PR TITLE
Datahubio 26 project structure

### PR DESCRIPTION
**Pull request by JanickiJ**

### CODE REVIEW

File: src/components/plot.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            <ResponsiveContainer width={1000} aspect={2.5}>
+                <LineChart data={plotData.points}>
+                    <YAxis/>
+                    <CartesianGrid stroke="#eee" strokeDasharray="5 5"/>
+                    <XAxis dataKey={"timestamp"} hide={false}/>
+                    <YAxis/>
+                    {
+                        Object.entries(plotData.charts).map(chart => {
+                            return (<Line key={chart[0]} type="monotone" dataKey={"values." + chart[0]}
+                                          stroke={chart[1].color}/>)
```
Comment (Singularity23x0) [inline src/components/plot.jsx]: Weird split of the line - for consistency sake you should maybe split all Line arguments into their own lines.

File: src/containers/NavbarContainer.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import {connect} from 'react-redux';
+import Navbar from '../components/navbar'
+import {readConfigFile} from '../utils/config'
+import {loadConfig} from '../actions/config'
+
+function mapStateToProps(state, ownProps) {
+    return {}
```
Comment (Singularity23x0) [inline src/containers/NavbarContainer.js:7]: Should it return an empty body? If it's not the final version I suggest a //TODO comment.

File: src/utils/plotData.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    constructor(title) {
+        this.title = title;
+    }
+
+    addPoint(timestamp, values) {
+        let diff = Object.keys(values)
+            .filter(v => !Object.keys(this.charts).includes(v));
+        if (diff.length) {
+            throw new Error(diff + " chart names not defined");
```
Comment (Singularity23x0) [inline src/utils/plotData.js]: "Magic strings" should not be used - extrac it to constant.

File: src/components/plot.jsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+import PropTypes from 'prop-types';
+
+function Plot({key, plotData}) {
+    return (
+        <div style={{display: 'flex', justifyContent: 'center', alignItems: 'center', height: '100vh'}}>
+            <ResponsiveContainer width={1000} aspect={2.5}>
+                <LineChart data={plotData.points}>
+                    <YAxis/>
+                    <CartesianGrid stroke="#eee" strokeDasharray="5 5"/>
+                    <XAxis dataKey={"timestamp"} hide={false}/>
```
Comment (Singularity23x0) [inline src/components/plot.jsx:12]: Many hardcoded "magic" values (like height and width) - I think we should extract them somewhere, unless there is a solid reason not to (in that case pls explain).

Comment (JanickiJ) [reply src/components/plot.jsx:12]: @StencelMichal

File: src/components/plot.jsx (hunk 1 above)
Comment (JanickiJ) [reply src/components/plot.jsx]: @StencelMichal

File: src/containers/NavbarContainer.js (hunk 2 above)
Comment (JanickiJ) [reply src/containers/NavbarContainer.js:7]: I It just example if anyone would want to add props to a component

File: src/utils/plotData.js (hunk 3 above)
Comment (JanickiJ) [reply src/utils/plotData.js]: @StencelMichal

**Pull request merged**